src/NewsService.Models.Db/DbNews.cs

```diff
@@ -11,14 +11,16 @@ public class DbNews
         public string Content { get; set; }
```

Comment:
Do you want to delete attribute and create a configuration class?

Reply:
that not in my task